### ISSUE TITLE
Gazebo isn't using NVidia card

### ISSUE THREAD

**Issue by mintar** [brean/mobipick_labs_docker#1]
I have a dedicated NVidia graphics card (running on Ubuntu 20.04). When I run the commands from the README, Gazebo inside the Docker container is not using the NVidia card.

You can see that visually by the missing lights and shadows in the Gazebo GUI. The following screenshots are from running Gazebo directly on my host system (not using the Docker image from this repo):

### 1. Intel graphics

![gazebo_intel](https://user-images.githubusercontent.com/320188/218471452-4019d78a-5171-43cb-ace2-35aca1f0ae42.png)

### 2. NVidia graphics

In order to activate the NVidia graphics card, I had to set the following environment variables:

```bash
## activate offload rendering for OpenGL apps
## see https://askubuntu.com/questions/1201072/how-nvidia-on-demand-option-works-in-nvidia-x-server-settings
export __NV_PRIME_RENDER_OFFLOAD=1
export __GLX_VENDOR_LIBRARY_NAME=nvidia
```

Result:

![gazebo_nvidia](https://user-images.githubusercontent.com/320188/218471721-679fb332-e492-4857-abe9-107fc0f93f5e.png)

When I start Gazebo inside Docker (as described in the README), the result looks similar to (1). I tried setting the environment variables in the `docker-compose.yml`, but it didn't work.

P.S.: Nice work on this repo, BTW!

**Comment by brean** [comment]
Thanks! Have you set the nvidia-container-runtime as your default runtime in `/etc/docker/daemon.json`?
 (as described here in step 4: https://docs.nvidia.com/dgx/nvidia-container-runtime-upgrade/index.html )
 otherwise you need to force the docker-compose file to use the "nvidia" runtime: `runtime: nvidia`

**Comment by mintar** [comment]
Unfortunately, editing `daemon.json` didn't help (same output as before).

Adding `runtime: nvidia` also didn't help because it produces the following error:

```
Error response from daemon: network with name mobipick already exists
non-network local connections being added to access control list
ERROR: The Compose file './docker-compose.yml' is invalid because:
Unsupported config option for services.mobipick: 'runtime'
Unsupported config option for services.mobipick_cmd: 'runtime'
non-network local connections being removed from access control list
```

FWIW, here's somebody that has the same problem (but no solution): https://forums.developer.nvidia.com/t/glxinfo-displays-nvidia-vendor-on-host-but-not-in-container/231906

**Comment by mintar** [comment]
I fixed it! See #2.
